pkg/cmd/openshift-kube-apiserver/openshiftkubeapiserver/flags.go

```diff
@@ -272,8 +274,12 @@ func admissionFlags(admissionPluginConfig map[string]configv1.AdmissionPluginCon
 	}
 	tempFile.Close()
 
+	// we need to list all of the DefaultOffPlugins minus the ones  that were explicitly enabled
+	// and explicitly disable those
+	forceOffWithDefaults := originadmission.DefaultOffPlugins.Difference(sets.NewString(forceOn...))
```

Comment:
Set this instead: https://github.com/kubernetes/kubernetes/blob/master/pkg/kubeapiserver/options/plugins.go#L65